src/dashboard/Data/Browser/BrowserTable.react.js

```diff
@@ -201,7 +201,7 @@ export default class BrowserTable extends React.Component {
             value = '';
           } else if (type === 'Array') {
             if (value) {
-              value = value.map(val => val instanceof Parse.Object ? val.toPointer() : val);
+              value = value.map(val => val instanceof Parse.Object ? val.toPointer() : (typeof val.getMonth === 'function' ? { __type: "Date", iso: val.toISOString() } : val));
```

Comment:
I would prefer not having nested ternary operators here. What do you think?
And when the user edits the date in the array, is it saved back correctly?

Reply:
Yes, it is saved correctly. Even the previous version was able to save provided `{"__type":"Date","iso":"2017-09-05T18:06:06.943Z" }` correctly as a valid Date, but it was then again displayed (after refresh) as `"2017-09-05T18:06:06.943Z"`. Now it is still displayed as the iso string, but when you open the cell, it adds this `{"__type":"Date","iso": .... }` so that it is saved correctly when update is confirmed :-)

How would you recommend to avoid nested ternary operators?

Reply:
I would do something like this: 

```
value = value.map(val => {
    if (val instanceof Parse.Object) {
        return val.toPointer;
    } else if (typeof val.getMonth === 'function') {
        return { __type: "Date", iso: val.toISOString() };
    }

    return val;
});
```